### ISSUE TITLE
html lang and body class missing

### ISSUE THREAD

**Issue by fritzmg** [contao/installation-bundle#15]
In the install tool, the `<html>` language and the `<body>` class are missing in Contao 4.1.3 and 4.2.0-beta1 for some reason:

``` html
<!DOCTYPE html>
<html lang="">
…
<body class="">
```

It appears the Twig variables `language` and `ua` are set with an empty string.

Interestingly, it works on the license accept page (see HTML source in #14).

Environment:
- Windows 10
- XAMPP 5.6.14
- PHP 5.6.14
- PHP running under CGI

**Comment by fritzmg** [comment]
This is because [here](https://github.com/contao/installation-bundle/blob/1.1.0-beta1/src/HttpKernel/InstallationKernel.php#L31)

``` php
if ($this->canBootRealSystem()) {
    parent::boot();
    $this->bootRealSystem();
} else {
    $this->initializeBundles();
    $this->bootHelperSystem();
}
```

`$this->bootHelperSystem();` will not be called anymore, once the `localconfig.php` has been created. And without `bootHelperSystem`, those global Twig variables are not set. This is because `bootHelperSystem` calls `ContainerFactory::create`, which in turn [adds these globals](https://github.com/contao/installation-bundle/blob/1.1.0-beta1/src/DependencyInjection/ContainerFactory.php#L114) to the Twig environment:

``` php
$twig = new \Twig_Environment($twigLoader);
$twig->addGlobal('path', $request->getBasePath());
$twig->addGlobal('language', str_replace('_', '-', $locale));
$twig->addGlobal('ua', Environment::get('agent')->class);
```

**Comment by leofeyer** [comment]
Fixed in 5c5e641ee0a47b67bde071185ea17e87cfeb7616.
